Dockerfile

```diff
@@ -1,15 +1,23 @@
+ARG ALPINE_VERSION=3.12
+ARG NODE_VERSION=15
+ARG NGINX_VERSION=1.19.6
+
 # multistage - builder image
-FROM node:alpine AS builder
+FROM node:${NODE_VERSION}-alpine${ALPINE_VERSION} AS builder
 WORKDIR /app/Lightspeed-react
-COPY . .
+# Layer caching for node modules
+COPY package.json package-lock.json ./
 RUN npm install
-
-# configure ip, hardcoded to webrtc container address (8080) for now
-RUN sed -i "s|stream.gud.software|localhost|g" public/config.json
+COPY . .
 
 # build it
```

Comment:
Comments are not necessary (lines 5, 8, 13, 16, etc). Separate each build stage with two line breaks.

Reply:
Agreed on that, didn't want to be nitpicking, thanks for leading the nitpicking way 😄